package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@trycourier/courier",
-  "version": "1.7.1",
+  "version": "1.7.2",
```

Comment:
If we are going to bump the version, please also include updates to the ChangeLog so we can push a new tag after this has been merged.

Reply:
Good point. Done!